docs/pages/getting-started.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Getting Started
+layout: default
+permalink: /getting-started
+nav_order: 2
+---
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+---
+
+# Getting Started
+
+Getting started requires a few steps:
+There are a number of steps that must be completed when first getting started with Stabilizer.
+1. Configure the firmware
+    * This requires updating any parameters, such as static IP addresses and
+   sampling rate.
+1. Build the application
+    * This requires compiling the code after configuration parameters have been
+   updated.
+1. Upload the application
+    * Once fimrware has been built, it needs to be programmed onto the device.
+1. Set up MQTT
+    * Stabilizer utilizes MQTT for telemetry and configuration.
+
+The following sections will walk you through completing each of these steps.
+
+# Program Stabilizer
```

Comment:
These sections here don't follow the path that was promised above. Also you need to first download (clone) the code before being able to edit `configuration.rs`.

Reply:
They do, but the docs are divided into two steps:
1. Get updated firmware on the device
2. Set up MQTT

That division isn't clear in the steps above. I'll look into a fix.